pom.xml

```diff
@@ -231,6 +231,19 @@
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-checkstyle-plugin</artifactId>
+                    <executions>
+                        <execution>
+                            <id>validate</id>
+                            <phase>validate</phase>
+                            <configuration>
+                                <suppressionsLocation>${checkstyle.suppressions.location}</suppressionsLocation>
+                                <suppressionsFileExpression>checkstyle.suppressions.file</suppressionsFileExpression>
+                            </configuration>
```

Comment:
We don't seem to be retaining some of the previous properties. I'm not sure, but I _think_ we want to keep (most) of them?

```xml
<includes>**\/*.java</includes> <!-- Makes it look at only .java code files, since it chokes on groovy files -->
<resourceIncludes>**/*.properties,**/*.sample</resourceIncludes> <!-- Look at some of the resource files -->
<includeTestSourceDirectory>true</includeTestSourceDirectory> <!-- Include test code -->
<consoleOutput>true</consoleOutput> <!-- Dump output to the console -->
```